docs/components/modeler/web-modeler/diagram-errors.md

```diff
@@ -0,0 +1,33 @@
+---
+id: diagram-errors
+title: Diagram errors
+description: This feature assists you in debugging and fixing errors in your processes.
+---
+
+### Lint errors
+
+Based on a set of debug rules, Web Modeler continuously validates implementation properties for a process diagram while the user is modeling. The debug rules are added to the debug panel at the bottom of Web Modeler. Expand the debug panel to view the errors.
```

Comment:
I would clarify here in the case someone cannot see the direct image. For example, "Expand the debug panel to view the errors by clicking the **Diagram errors** header."

Given the following section alludes to this, you could say "Expand the debug panel to view the errors by clicking the **Diagram errors** header. The panel is collapsed by default and the latest state (expanded or collapsed) is remembered for the next time you open Web Modeler." Then, you could just remove the `### Expand and collapse` section as I don't think we need a section dedicated to this.